tools/add-copyright-header.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+scriptdir="$(cd $(dirname $0) && pwd)"
+
+for i in $(git ls-files | grep '.ts$'); do
+  if ! grep -q Copyright $i
```

Comment:
Is there a way to tell grep to only look at the first line? As this would ignore files that contain the string `Copyright` somewhere else too, right?